### PR TITLE
adding the sap blog

### PR DESCRIPTION
**Pull request by rahuljo**

### CODE REVIEW

File: docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+slug: dlt-dagster
```
Comment (matthauskrzykowski) [inline docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md:2]: should be a different slug

File: docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+title: "Combining SAP HANA and Snowflake using dlt"
```
Comment (matthauskrzykowski) [inline docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md:3]: suggestion: Exploring data replication of SAP HANA to Snowflake using dlt

File: docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+But what about enterprise systems like SAP? They are, after all, the most ubiquitous tooling out there: according to SAP [data](https://assets.cdn.sap.com/sapcom/docs/2017/04/4666ecdd-b67c-0010-82c7-eda71af511fa.pdf), 99 out of 100 largest companies are SAP customers. A huge part of the reason for this is that their ERP system is still the gold standard in terms of effectivity and reliability. However, when it comes to OLAP workloads like analytics, machine learning, predictive modelling etc., [many companies prefer the convenience and cost savings of modern cloud solutions](https://www.statista.com/statistics/967365/worldwide-cloud-infrastructure-services-market-share-vendor/) like GCP, AWS, Azure, etc..
+
+So, wouldn’t it be nice to be able to integrate SAP into the modern ecosystem?
+
+Unfortunately, this is not that simple. Unlike modern tools, SAP does not integrate easily with non-SAP systems, and migrating data out from SAP is complicated and/or costly. This often means that ERP data stays separate from analytics data, resulting in data silos and preventing companies from making the most out of their analytics.
```
Comment (matthauskrzykowski) [inline docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md:24]: delete "Unlike modern tools"

delete "resulting in data silos and preventing companies from making the most out of their analytics."

File: docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+## Creating a dlt integration  
+  
+As a first step, I decided to create a custom dlt integration to SAP’s in-memory data warehouse: SAP HANA. I chose SAP HANA since it’s an OLAP database, and our users have been specifically asking us for this connector. Plus, given its SQL backend and [Python API](https://developers.sap.com/tutorials/hana-clients-python.html), dlt should also have no problem connecting to it.
```
Comment (matthauskrzykowski) [inline docs/website/blog/2023-11-29-combining-sap-hana-and-snowflake-using-dlt.md:28]: change into: Our users have been asking for SAP HANA data, hence I decided to create a custom dlt integration to SAP’s in-memory data warehouse:

**Review by matthauskrzykowski — CHANGES_REQUESTED** [review]
see the 4 comments

**Comment by rahuljo** [comment]
@matthauskrzykowski Made the updates. Please approve if everything looks good

**Review by matthauskrzykowski — APPROVED** [review]

**Pull request merged**